<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEdit</class>
 <widget class="QMainWindow" name="UserEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户信息/权限设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户信息编辑</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>邮箱</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电话</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UID</string>
    </property>
   </widget>
   <widget class="QLabel" name="uidLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phoneEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>角色</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="userRoleBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="admin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>系统管理员</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="manager">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>库存经理</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="hr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>人力资源经理</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="user">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>销售员 / 普通用户</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="userPermissionBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>用户权限编辑</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>43</y>
       <width>63</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>权限域</string>
     </property>
    </widget>
    <widget class="QComboBox" name="permissionDomain">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>83</y>
       <width>63</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>权限类型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="permissionType">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>153</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>操作</string>
     </property>
    </widget>
    <widget class="QComboBox" name="permisionOp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="savePermissionButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>85</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>修改权限</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>63</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>当前状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="statusLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>651</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>说明：1. 系统管理员的操作没有任何限制；但只有 UID = 1 的管理员能够创建和删除其它系统管理员。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2. 库存经理默认对商品及库存等具有写权限；人力资源经理默认对用户信息表具有写权限。</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3. 普通用户仅对商品及库存表具有读权限和售货表的写权限，其它权限需要动态编辑。</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>500</y>
      <width>85</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastLoginTitleTable">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>63</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>上次登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastLoginLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
